### PR TITLE
New feature list orders

### PR DESCRIPTION
**Pull request by bf2285**
- List all orders function tested and implemented

- Modified `factories.py` and `test_models.py` according to `make lint`

### CODE REVIEW

**Comment by MahamArif** [comment]
Please connect the corresponding user story to your PR.

File: service/routes.py [hunk 1]
```diff
@@ -64,6 +64,18 @@ def get_order(order_id):
...
 
+######################################################################
+# List Orders
+######################################################################
+@app.route("/orders", methods=["GET"])
+def list_orders():
+    """List all of the Orders"""
+    app.logger.info("Request for order list")
+    orders = Order.all()
+    results = [order.serialize() for order in orders]
```
Comment (MahamArif) [inline service/routes.py:75]: Please add an info log here to indicate how many records are returned by this API response:

`app.logger.info("Returning %d orders", len(results))`

File: tests/test_routes.py [hunk 2]
```diff
@@ -176,3 +176,11 @@ def test_create_order_bad_status(self):
         test_order["status"] = "created"  # wrong value
         response = self.app.post(BASE_URL, json=test_order)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_list_orders(self):
+        """ It should list orders"""
+        self._create_orders(5)
+        response = self.app.get(BASE_URL)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.get_json()
```
Comment (MahamArif) [inline tests/test_routes.py:185]: Please move this test to the happy tests section above.

File: tests/test_routes.py [hunk 3]
```diff
@@ -176,3 +176,11 @@ def test_create_order_bad_status(self):
...
+        self.assertEqual(len(data), 5)
```
Comment (MahamArif) [inline tests/test_routes.py:186]: Please run `make lint` to fix any lint issues introduced with your code. For instance, you have not added a new line to the end of the file.

Comment (bf2285) [reply tests/test_routes.py:186]: Running linting...
flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
0
flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
./service/routes.py:7:1: F401 'flask.Flask' imported but unused
./tests/test_routes.py:11:1: F401 'unittest.mock.MagicMock' imported but unused
./tests/test_routes.py:11:1: F401 'unittest.mock.patch' imported but unused
3     F401 'flask.Flask' imported but unused
3
make: *** [Makefile:31: lint] Error 1

Do you want me to get rid of the unused?

File: tests/test_routes.py (hunk 2 above)
Comment (bf2285) [reply tests/test_routes.py:185]: done

File: service/routes.py (hunk 1 above)
Comment (bf2285) [reply service/routes.py:75]: done

File: tests/test_routes.py (hunk 3 above)
Comment (MahamArif) [reply tests/test_routes.py:186]: I think leave the unused imports for now because they may be used later.

Comment (bf2285) [reply tests/test_routes.py:186]: I removed them to see further reports on the formatting. We can always add it if we need it in the future.

**Pull request merged**